src/main/java/step2/RacingGame.java

```diff
@@ -52,14 +52,19 @@ public int getMaxPosition(List<Car> carCollection) {
         int maxPosition = 0;
 
         for (Car car : carCollection) {
-            if (car.getPosition() > maxPosition) {
-                maxPosition = car.getPosition();
-            }
+            maxPosition = findMaxPosition(car, maxPosition);
         }
 
         return maxPosition;
     }
 
+    public int findMaxPosition(Car car, int maxPosition) {
+        if (car.getPosition() > maxPosition) {
+            maxPosition = car.getPosition();
+        }
+        return maxPosition;
```

Comment:
```suggestion
        return  (car.getPosition() > maxPosition) ? car.getPosition() : maxPosition;
```

이 부분은 삼항연산자를 활용할 수 있을거 같아요

그리고 이 메서드는 내부에서만 사용하는데 접근제어자를 private으로 두어도 되지 않을까요